### DISCUSSION TITLE
[firebase_dynamic_links] Social Sharing Previews not working on Gettr

### DISCUSSION THREAD

**Discussion by KennyChan94P** [firebase/flutterfire#7279] [Q&A]
i managed to create a dynamic link with social sharing preview (title, description and image) and the details are shown when i share the link as a post on Facebook and as a tweet on Twitter.
but the link does not work in Gettr. when i share the link as a post on Gettr, it shows as a string but without the (title, description and image).
does anyone have any clue on how to show the (title, description and image) of the dynamic link on Gettr?
thank you.

**Comment by Vaytrex** [comment]
For anyone landing here after the Aug 2025 shutdown: there's no drop-in. You need a URL that branches on User-Agent (App Store / intent:// with browser_fallback_url / website) and skips redirecting crawlers so previews survive. Full write-up with the intent format: https://linkbah.app/firebase-dynamic-links-alternative
